test/test_ops.py

```diff
@@ -89,33 +89,33 @@ def _gradgrad_test_helper(self, device, dtype, op, variant):
         return self._check_helper(device, dtype, op, variant, 'gradgradcheck')
 
     # Tests that gradients are computed correctly
-    @dtypes(torch.double, torch.cdouble)
+    @dtypes(torch.double)
```

Comment:
Why are these removed? Are you planning to add them bad later?
If so, there is a missing TODO, otherwise, you can just remove this and use `torch.double` as the dtype no?

cc @mruberry that might have some comments on the testing.

Reply:
added them back